mms/benchmarks/benchmark.py

```diff
@@ -422,7 +427,6 @@ def run_benchmark():
 
     parser.add_argument('-d', '--docker', nargs=1, type=str, default=None, help='Docker hub path to use')
     parser.add_argument('-i', '--input', nargs=1, type=str, default=None, help='The input to feed to the test')
-    parser.add_argument('-g', '--gpus', nargs=1, type=int, default=None, help='Number of gpus.  Leave empty to run CPU only')
```

Comment:
You still need this line.  It sets GPU vs CPU mode and pargs.gpus

Reply:
Ok.